packages/react-querybuilder/src/defaults.ts

```diff
@@ -42,6 +42,8 @@ export const defaultPlaceholderOperatorGroupLabel = placeholderLabel;
  */
 export const defaultJoinChar = ',';
 
+export const defaultQuoteChar: `'` | `"` = `'`;
```

Comment:
We don't need to export this or even define it at all. Just default it to `'` in `formatQuery`.